src/qililab/data_management.py

```diff
@@ -174,7 +173,7 @@ def save_platform(path: str, platform: Platform) -> str:
 
 
 def build_platform(
-    runcard: str | dict | None = None, path: str | None = None, connection: API | None = None, new_drivers: bool = False
+    runcard: str | dict | None = None, path: str | None = None, connection=None, new_drivers: bool = False
```

Comment:
Shouldn't we delete `connection` completely?

Reply:
Yess indeed, and also remove all the test related to connection and live plotting too! 
Was just testing 🤓

Reply:
Yeah was my bad, sry 🙏